File: TechBlazers-Part1/lottery.py
from random import randint

lottery_number = randint(100, 999)
users_input = int(input("Enter a three-digit number: "))

lottery_digit1 = lottery_number % 10
lottery_number = lottery_number // 10

lottery_digit2 = lottery_number % 10
lottery_number = lottery_number // 10

lottery_digit3 = lottery_number

users_digit1 = users_input % 10
users_input = users_input // 10

users_digit2 = users_input % 10
users_input = users_input // 10

users_digit3 = users_input

if users_digit3 == lottery_digit3 and users_digit2 == lottery_digit2 and users_digit1 == lottery_digit1:
	print("Your award is $10,000")
elif (users_digit3 == lottery_digit3 or users_digit2 == lottery_digit2 or users_digit1 == lottery_digit1) and (users_digit2 == lottery_digit1 or users_digit2 == lottery_digit2 or users_digit2 == lottery_digit3) and (users_digit3 == lottery_digit1 or users_digit3 == lottery_digit2 or users_digit3 == lottery_digit3):
	print("Your award is $3,000")
elif (users_digit3 == lottery_digit3 or users_digit2 == lottery_digit2 or users_digit1 == lottery_digit1) or (users_digit2 == lottery_digit1 or users_digit2 == lottery_digit2 or users_digit2 == lottery_digit3) or (users_digit3 == lottery_digit1 or users_digit3 == lottery_digit2 or users_digit3 == lottery_digit3):
	print("Your input matches one digit in the lottery number, your award is $0")
else:
	print("Sorry!!!, your input do not match the lottery number. Try again next time")

File: TechBlazers-Part1/Lottery.java
import java.util.Scanner;
import java.util.Random;

public class Lottery {
    public static void main(String[] args) {
        Random random = new Random();
        int lotteryNumber = random.nextInt(900) + 100;

        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a three-digit number: ");
        int usersInput = scanner.nextInt();

        int lotteryDigit1 = lotteryNumber % 10;
        lotteryNumber /= 10;

        int lotteryDigit2 = lotteryNumber % 10;
        lotteryNumber /= 10;

        int lotteryDigit3 = lotteryNumber;

        int usersDigit1 = usersInput % 10;
        usersInput /= 10;

        int usersDigit2 = usersInput % 10;
        usersInput /= 10;

        int usersDigit3 = usersInput;

        if (usersDigit3 == lotteryDigit3 && usersDigit2 == lotteryDigit2 && usersDigit1 == lotteryDigit1) {
            System.out.println("Your award is $10,000");
        } else if ((usersDigit3 == lotteryDigit3 || usersDigit2 == lotteryDigit2 || usersDigit1 == lotteryDigit1) &&
                   (usersDigit2 == lotteryDigit1 || usersDigit2 == lotteryDigit2 || usersDigit2 == lotteryDigit3) &&
                   (usersDigit3 == lotteryDigit1 || usersDigit3 == lotteryDigit2 || usersDigit3 == lotteryDigit3)) {
            System.out.println("Your award is $3,000");
        } else if ((usersDigit3 == lotteryDigit3 || usersDigit2 == lotteryDigit2 || usersDigit1 == lotteryDigit1) ||
                   (usersDigit2 == lotteryDigit1 || usersDigit2 == lotteryDigit2 || usersDigit2 == lotteryDigit3) ||
                   (usersDigit3 == lotteryDigit1 || usersDigit3 == lotteryDigit2 || usersDigit3 == lotteryDigit3)) {
            System.out.println("Your input matches one digit in the lottery number, your award is $0");
        } else {
            System.out.println("Sorry!!!, your input does not match the lottery number. Try again next time.");
        }
    }
}
